mlir/test/Dialect/Shape/canonicalize.mlir

```diff
@@ -86,6 +86,19 @@ func.func @broadcast() -> !shape.shape {
 
 // -----
 
+// Variadic case including extent tensors.
```

Comment:
I believe now the single operand broadcast is also supported, could you add test for it?